frontend/js/src/year-in-music/2022/YearInMusic.tsx

```diff
@@ -1191,7 +1191,11 @@ export default class YearInMusic extends React.Component<
               rel="noopener noreferrer"
             >
               on twitter
-            </a>
+            </a>{" "}
```

Comment:
Wow, that is special.

Reply:
welcome to react and code autoformatters